app/components/ui/Modal/Modal.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+import ReactModal from 'react-modal';
+
+import { icon } from 'assets';
+
+import styles from './Modal.module.scss';
+/**
+ * This modal component is used on the service/search results page. It provides
+ * shared layout and shared styling.
+*/
+
+export const Modal = ({
+  isOpen, closeModal, children, addModalClass,
+}: {
+  isOpen: boolean;
+  closeModal: () => void;
+  children: ReactNode;
+  addModalClass?: string;
+}) => (
+  <ReactModal
+    isOpen={isOpen}
+    className={`${styles.modal} ${addModalClass}`}
+    overlayClassName={styles.modalOverlay}
+    onRequestClose={closeModal}
+  >
+    <div role="button" tabIndex={0} onClick={closeModal}>
+      <img src={icon('close')} alt="close" className={styles.closeModalIcon} />
+    </div>
```

Comment:
Why not use an actual `<button>` instead of a `<div>` that mimics a button? I'm guessing you copied this from the existing modals on the app, so we probably don't need to address this here, and instead we can probably take of this in a general cleanup of all the modals on the site to use a single set of base styles.